### DISCUSSION TITLE
No public gene lists are displayed in local installation

### DISCUSSION THREAD

**Discussion by GeorginaAmbriz** [broadinstitute/seqr#3333] [General]
Hello,
I'm using a local installation of seqr. I've uploaded an exome trio dataset and added the family and pedigree information. Now, I want to run a variant search, but when I open the `Add Gene List ` link, I don't see any public list in the `Public Gene Lists` section.

Aren't public list supposed to be displayed ? Do I have to upload a public list like Clinvar separately? 
What should I do to add public lists?

Thank you for your support,
Georgina Jimenez
Bioinformatics core
Montreal Neurological Institute

**Comment by hanars** [comment]
Public gene lists just means gene lists that someone has created in seqr and allowed to be made available to other users. Someone in your seqr instance needs to create them manually, they are not shared across instances of seqr. In general, gene lists can be very specific to individual groups/analysts depending on what areas of interest they have, whether they focus on novel gene discovery or known gene diagnosis, etc. so theres no real source of truth for gene lists. 

If you are interested in using panel app lists (https://panelapp.genomicsengland.co.uk/) you can add those by running 
```
docker-compose exec seqr manage.py import_all_panels https://panelapp.agha.umccr.org/api/v1 --label=AU
docker-compose exec seqr manage.py import_all_panels https://panelapp.genomicsengland.co.uk/api/v1 --label=UK
```

**Comment by GeorginaAmbriz** [comment]
Hello Hanars,
Thank you for answering. I tried to add the panel app list by running the commands you gave. When I ran either of both commands I had the following error message:
```
$  docker-compose exec seqr manage.py import_all_panels https://panelapp.genomicsengland.co.uk/api/v1 --label=UK
OCI runtime exec failed: exec failed: unable to start container process: exec: "manage.py": executable file not found in $PATH: unknown
```
How can I fix this?
Thank you again for your support.

**Reply by hanars** [reply]
hmm try `docker-compose exec seqr python -u manage.py import_all_panels https://panelapp.agha.umccr.org/api/v1 --label=AU`
